backend/app/models/__init__.py

```
@@ -21,6 +21,8 @@
 from app.models.upload import FileMetadata
 from app.models.user import User, UserCreate, UserPublic, UserRole, UserUpdate
 
+from .user import EmailVerificationToken, PasswordResetToken
```

Ruff (0.11.9)

`.user.EmailVerificationToken` imported but unused; consider removing, adding to `__all__`, or using a redundant alias

(F401)